runtime/caml/startup_aux.h

```diff
@@ -36,6 +36,7 @@ extern uintnat caml_init_custom_major_ratio;
 extern uintnat caml_init_custom_minor_ratio;
 extern uintnat caml_init_custom_minor_max_bsz;
 extern uintnat caml_trace_level;
+extern uintnat caml_init_alloc_size;
```

Comment:
If this variable is only meant for addrmap serialization tables, maybe the name could be more specific? What about `caml_init_intern_addrmap_size`?